metricbeat/module/jolokia/jmx/_meta/docs.asciidoc

```diff
@@ -1,58 +1,67 @@
-This is the `jmx` metricset of the Jolokia module.
+The `jmx` metricset collects metrics from
+https://jolokia.org/reference/html/agents.html[Jolokia agents].
 
 [float]
 === Features and configuration
 Tested with Jolokia 1.3.4.
```

Comment:
So should this say 1.5.0 because it seems to contradict the earlier statement about the module.

Reply:
Yes, this should be 1.5.0 too.